code_generator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -172,10 +177,33 @@ namespace <#= qualifiedNamespace #>
                 }
 <# } #>
                 componentData.DirtyBit = false;
+
 <# if (componentDetails.IsBlittable) { #>
-                view.UpdateComponent(entity, componentData, UpdatesPool);
+                view.SetComponentData(entity, componentData);
```

Comment:
why did we change this line?

Reply:
The view previously provided the following API:
- UpdateComponent()
- UpdateComponentObject()
which both update the component snapshot and add a message to their respective reactive components.

In the new solution we update the component snapshot separately from adding the reactive message. This is also more in line with how we are doing events, commands, auth changes.